src/main/java/net/minecraftforge/common/BiomeManager.java

```diff
@@ -103,6 +105,11 @@ public static void addBiome(BiomeType type, BiomeEntry entry)
         if (list != null) list.add(entry);
     }
 
+    public static void addNetherBiome(Biome biome)
```

Comment:
Lacking documentation, when should this be called? I assume you intended `FMLCommonSetupEvent`?